tests/bsim/bluetooth/host/att/pipeline/test_scripts/run_test_shall_not_pipeline_variant_rx_tx_prio_invert.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+# Copyright 2023-2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+#  Test purpose:
+#
+#  Basic check that the DUT GATT client does not pipeline ATT
+#  requests.
+#
+#  Sending a new request on a bearer before the response to the
+#  previous request has been received ('pipelining') is a
+#  protocol violation.
+#
+#  Test variant 'rx_tx_prio_invert':
+#
+#  - The priority of the RX and TX threads is inverted compared
+#    to the default configuration. The result shall be the same.
+#
+#  Test procedure:
+#
+#  - DUT is excercised by calling `gatt_write` in a loop.
+#  - Tester does not immediately respond but delays the response
+#    a bit to ensure the LL has time to transport any extra
+#    requests, exposing a bug.
+#  - Tester verifies there is no such extra request while it's
+#    delaying the response. Detecting an extra request proves a
+#    protocol violation.
+
+source ${ZEPHYR_BASE}/tests/bsim/sh_common.source
+
+dut_exe="bs_${BOARD_TS}_tests_bsim_bluetooth_host_att_pipeline_dut_prj_conf"
```

Comment:
Just for info: you can break strings with \
and in this case you can also replace "tests_bsim_bluetooth_host_att_pipeline" with $(guess_test_long_name)

And a nit: Do you really need this long test script names and sim_ids? (the "run_test" part seems quite redundant